### DISCUSSION TITLE
Question about how the architecture graphics in the lecture were created

### DISCUSSION THREAD

**Discussion by LaurenzV** [sebastian-hofstaetter/teaching#103] [General]
Hello,

I was wondering whether you could tell me the program that was used to create some of the graphics that sometimes appear in the lecture/papers. I'm for example talking about the picture that shows the architecture of the TK model in the slides, but also the paper of the last exam contained a figure with a similar style, so I presume that it was made using the same program. I think they look pretty nice, so it would be great to know what program I can use to create similar ones.

Kind regards,
Laurenz

**Comment by sophiaalthammer** [comment]
Hi Laurenz,

thanks I also think they look really nice. They are created using Microsoft Visio!

Best,
Sophia
